docker-compose.yml

```diff
@@ -48,21 +48,6 @@ services:
       - REDIS_URL=storage:6379
       - SIGN_KEY
 
-  catalogue:
```

Comment:
Does it add any value to run it in docker/docker compose vs just running it directly when it's a single service?

Reply:
Maybe you are right. I just thought it would be nice to have it somewhere in docker-stuff to show it's part of the application, but it probably doesn't give much value.
So yep, I am fine with skipping.